### DISCUSSION TITLE
Rerender the rowHeight

### DISCUSSION THREAD

**Discussion by RyanZwr** [nick-keller/react-datasheet-grid#239] [Q&A]
When I change the  the input value of "DynamicDataSheetGrid",  I want to rerender the rowHeight according to my new input each time, how can I do that?

**Comment by nick-keller** [comment]
Dynamic row height is still in beta and undocumented, this is an issue I have not fixed yet :/

**Reply by akshaytanikatech** [reply]
Hey @nick-keller, is there any way i can achieve this? i've tried below and i want dynamic height of row to use multiple dropdown in cell or if there is any other way?

```
<DataSheetGrid
      // ref={datasheetRef}
      rowHeight={({rowData}) => {
      return { top: 60, height: 60 }; // custom function
      // return 40;  //  tried this also
      }}
/>
```

If you could help it would be great, thanks!
